check-update-security.sh

```diff
@@ -195,12 +195,12 @@ command_status() {
 max_retries=20
 success=0
 for ((r = 0; r < max_retries; r++)); do
+    sleep 5
```

Comment:
This was done because there was a very small corner case where the command invocation is seems to not exist at the first check. Instead of risking that chance, we wait 5 seconds. None of the tests after this change procured such the same issue.

Reply:
You mean `aws` command does not exist on the host but then appears later?

Reply:
nope, I mean when we invoke the _command on the instance_ by ssm, trying to check the _command invocation_ using comand_status() right away gives an error of not being able to find such an invocation on the instance. Perhaps it is not completely processed when the first aws call returns and not available until a few seconds pass.

Reply:
Okay thanks for explaining.